src/parser.test.js

```diff
@@ -0,0 +1,90 @@
+import parser from './parser';
+import { renderIntoDocument } from '../tests/utils/dom-render';
+
+const setup = () => {
+  return renderIntoDocument(`
+    <div>
+      <button>btn1</button>
+      <button>btn2</button>
+      <span>I'm a label</span>
+      <input placeholder="I'm a placeholder" />
+    </div>`);
+};
+describe('parser', () => {
+  it('should parse query successfully', () => {
+    const container = setup();
+    const result = parser.parse({
+      htmlRoot: container,
+      js: 'screen.getByText("I\'m a label")',
+    });
+    expect(result).toMatchSnapshot();
```

Comment:
I think using inline snapshots would be better instead

https://kentcdodds.com/blog/effective-snapshot-testing